simple-git-hooks.js

```diff
@@ -84,15 +84,13 @@ function getProjectRootDirectoryFromNodeModules(projectPath) {
     const projDir = projectPath.split(/[\\/]/) // <- would split both on '/' and '\'
 
     if (projDir.includes('.pnpm')
-        && projDir.length > 5
-        && _arraysAreEqual(projDir.slice(projDir.length - 5), [
+        && projDir.length > 3
+        && _arraysAreEqual(projDir.slice(projDir.length - 3), [
```

Comment:
Maybe we do not need that strong condition? Like, if we see `.pnpm` in our path, than user is using `.pnpm` for sure => resolve folder

Reply:
but it's LGTM, and a fix, so merging and releasing :)